docs/reference/commandline/dockerd.md

```diff
@@ -75,7 +75,8 @@ Options:
       --registry-mirror value                 Preferred Docker registry mirror (default [])
       --seccomp-profile value                 Path to seccomp profile
       --selinux-enabled                       Enable selinux support
-      --shutdown-timeout=15                   Set the shutdown timeout value in seconds
+      --shm-size int                          Set the default shm size for containers (default 67108864)
```

Comment:
@flx42 The human readable memory size (like `128m`) would be tricky for daemon config.

Currently the processing of `docker run` and `dockerd` are different. The `dockerd` could either takes a flag like `--default-shm-size=67108864` or allows the user to specify it in a json file like:
```
{
"default-shm-size" : 67108864
}
```

The flag form in `dockerd` could be processed like `docker run` to take a human readable memory size. 

However, `dockerd` will directly unmarshal the json config file into a struct `Config`. As a result, if we use string (for human readable memory size) for `default-shm-size`, it is not possible to convert it into `int` and store it in `daemon.configStore` (unless we change the current way of storing config in docker daemon).

I think such a change probably could be done in a separate PR as it involves much bigger impact. Would also like to be confirmed by docker maintainers before making the change.